ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -161,19 +161,16 @@ private[commands] class CommandTracker[Context]
         }
       }
 
+      import CommandTracker.nonTerminalCodes
+
       private def handleSubmitResponse(submitResponse: Ctx[(Context, String), Try[Empty]]) = {
         val commandId = submitResponse.context._2
         submitResponse.value.fold(
           {
-            case GrpcStatusError(status)
-                if !CommandTracker.nonTerminalCodes.contains(status.getCode) =>
+            case GrpcException(GrpcStatus(code, description), _) if !nonTerminalCodes(code) =>
               val output = getOutputForTerminalStatusCode(
                 commandId,
-                Status(
-                  status.getCode.value(),
-                  if (status.getDescription == null) ""
-                  else status.getDescription,
-                  Nil))
+                Status(code.value(), description.getOrElse(""), Nil))
```

Comment:
Looks like `Nil` (or rather `Seq.empty`, which is the same thing) is the default value for the `details` parameter and can be omitted.

```suggestion
                Status(code.value(), description.getOrElse("")))
```